app/build.gradle

```diff
@@ -249,7 +249,7 @@ android {
 
     defaultConfig {
         minSdkVersion 21
-        targetSdkVersion 31
+        targetSdkVersion 33
```

Comment:
@OrangeAndGreen We can't change the targetSdkVersion without actually making the supporting Android 33 changes that Ahmad is [working on](https://github.com/dimagi/commcare-android/pull/2683).  I think we should only be able to change the `compileSdkVersion` for incorporating the navigation UI component in code.